### PR TITLE
DDF-3297 Update policy for Felix Webconsole

### PR DESCRIPTION
**Pull request by pklinef**
#### What does this PR do?
Added permissions needed to start and use the Felix Web Console.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@rymach 
@SmithJosh 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
@codice/security

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@bdeining
@stustison

#### How should this be tested? (List steps with links to updated documentation)
Start the `webconsole` feature.  Verify that https://localhost:8993/system/console/ works and that all pages in the menu work.

#### What are the relevant tickets?
[DDF-3297](https://codice.atlassian.net/browse/DDF-3297)

#### Checklist:
- [ n/a ] Documentation Updated
- [ n/a ] Update / Add Unit Tests
- [ n/a ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by rymach** [comment]
Hero'd - no issues. Thanks for this fix.

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 1]
```diff
@@ -306,6 +306,22 @@ grant codeBase "file:/org.apache.cxf.cxf-rt-bindings-soap/org.apache.cxf.cxf-rt-
     permission java.security.SecurityPermission "insertProvider";
     permission java.io.FilePermission "${ddf.home.perm}etc${/}application-definitions", "read";
     permission java.io.FilePermission "${ddf.home.perm}etc${/}application-definitions${/}*", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.shell.cfg", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.management.cfg", "read,write";
```
Comment (SmithJosh) [inline distribution/ddf-common/src/main/resources/security/default.policy]: ❓ How many of the bundles in this block need these permissions?

Comment (stustison) [reply distribution/ddf-common/src/main/resources/security/default.policy]: I would guess that all or nearly all of them do. These bundles are the standard chain of bundles that get hit on your way through Jetty and CXF.

**Comment by stustison** [comment]
build now

**Pull request merged**